addOns/alertFilters/src/main/java/org/zaproxy/zap/extension/alertFilters/AlertFilterAPI.java

```diff
@@ -298,11 +330,59 @@ public ApiResponse handleApiAction(String name, JSONObject params) throws ApiExc
                 }
                 return ApiResponseElement.FAIL;
 
+            case ACTION_APPLY_ALL:
+                Map<String, Integer> applyCountsMap = new HashMap<>(2);
```

Comment:
You'd have to also set the load factor to avoid re-allocations.

Reply:
I believe it should be fine. 1/2 = 0.5, which is less than the default of 0.75 so no re-allocation should be necessary. (Or I'm totally missing something)

https://www.javatpoint.com/load-factor-in-hashmap

Reply:
It's adding two elements, so should be 2/2.

Reply:
Will tweak it shortly